codes/classical/bits/hamming.yml

```diff
@@ -0,0 +1,40 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: hamming
+physical: bits
+logical: bits
+
+name: 'Hamming code'
+#introduced: '\cite{doi: 10.1002/j.1538-7305.1950.tb00463.x.}'
+
+description: 'Hamming Codes are a family of linear codes with parameters \((2^r-1,2^r-r-1, 3)\) for any
```

Comment:
Part about error detection should be moved to protection field and removed here.